sklearn/metrics/pairwise.py

```diff
@@ -1015,19 +1015,35 @@ def paired_euclidean_distances(X, Y):
 
 
 def paired_manhattan_distances(X, Y):
-    """Compute the L1 distances between the vectors in X and Y.
+    """Compute the paired L1 distances between X and Y.
+
+    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), ...,
+    (X[n_samples], Y[n_samples]).
```

Comment:
I think I am off by 1 here -- if `shape(X) == shape(Y) == (n_samples, n_features) == (2, 2)`, then distances are calculated for `(X[0], Y[0]), (X[1], Y[1])`. Do you agree @jjerphan?

```suggestion
    (X[n_samples - 1], Y[n_samples - 1]).
```

Reply:
That's right.